server/modules/worker.js

```diff
@@ -1,10 +1,15 @@
 import client from '../helpers/redis';
 
+import { onboardingJobs } from '../jobs';
+
 export default {
   init: () => {
     client.set('startTime', JSON.stringify(new Date()));
   },
   exec: () => {
+    // execute onboading jobs
+    onboardingJobs();
```

Comment:
You also don't need to add this. In fact there is no need for any change to the `worker.js` file, the logic of executing the jobs has been handled in another PR that is already merged.